urllib3/util/ssl_.py

```diff
@@ -195,6 +197,16 @@ def create_urllib3_context(ssl_version=None, cert_reqs=ssl.CERT_REQUIRED,
     """
     context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
 
+    if hasattr(context, 'urllib3_wrapper_context'):
```

Comment:
Yeah, this is likely a bit better, but tricky to actually get to test. To test it I need to get the SSL module to raise an import error on the SSLContext. Any ideas how I go about that?